src/web/src/pages/layouts/MDXPageBase.tsx

```diff
@@ -23,16 +22,20 @@ const useStyles = makeStyles((theme) => {
       '& h1': {
         color: theme.palette.text.secondary,
         fontSize: 24,
-        paddingLeft: '5px',
+        paddingLeft: '150px',
+        paddingRight: '150px',
       },
       '& h2': {
         color: theme.palette.text.secondary,
         fontSize: 20,
-        paddingLeft: '5px',
+        paddingLeft: '150px',
+        paddingRight: '150px',
       },
       '& p': {
         color: theme.palette.text.primary,
         fontSize: 16,
+        paddingLeft: '150px',
+        paddingRight: '150px',
         padding: '3px 5px',
```

Comment:
You should combine these into `padding: '3px 150px'`. The current 5px padding is conflicting with what you're adding and is not needed.